docs/spec/dataclasses.rst

```diff
@@ -365,6 +365,9 @@ positional and may use other names.
 * ``alias`` is an optional str parameter that provides an alternative
   name for the field. This alternative name is used in the synthesized
   ``__init__`` method.
+* ``converter`` is an optional parameter that specifies a
+  callable used to convert values when assigning to the associated
+  attribute.
```

Comment:
Wording nit: "associated attribute" feels like very specific terminology, but it's not used in describing any of the other field specifier parameters. If there is something particularly unique to `converter` that we are trying to communicate with this wording, maybe we should communicate it more explicitly? Otherwise, it looks more consistent to just use "field" rather than "associated attribute."
```suggestion
  callable used to convert values when assigning to the field.
```